R/dv8.R

```diff
@@ -4,28 +4,28 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-#' Exports DV8 clustering as a separate JSON file
+#' Transforms a DV8 binary cluster file to a json cluster file.
 #'
 #' Converts a clustering binary in the dv8 format *-clsx.dv8-cslx into a
 #'  *-clsx.json.
```

Comment:
I see you actually already included the dv8 format of the files, so it is clear to the user what we are calling a binary file. Ok!